src/websocket/server.cc

```diff
@@ -146,6 +146,14 @@ future<> connection::read_http_upgrade_request() {
         if (upgrade_header != "websocket") {
             throw websocket::exception("Upgrade header missing");
         }
+
+        sstring subprotocol = req->get_header("Sec-WebSocket-Protocol");
```

Comment:
Comment about the commit message (since I can't do it there): You called this an "HTML Header". I guess you meant "WebSocket header", or "HTTP header" (which is a bit weird since this is not HTTP, but it's true that they look the same)?